pennylane_lightning/core/src/gates/Gates.hpp

```diff
@@ -449,6 +470,7 @@ static auto getCRZ(T angle) -> std::vector<ComplexT<T>> {
             ZERO<ComplexT, T>(),
             ZERO<ComplexT, T>(),
             ZERO<ComplexT, T>(),
+            ZERO<ComplexT, T>(),
```

Comment:
Was there a bug in the implementation of `gateCRZ`? How's this not captured before? :)

Reply:
This module is mostly used for tests, and many gates aren't used in any test. I've caught some other bugs when implementing generic controls (i.e. DoubleExcitation gates defined as transposed of themselves, etc.) because I started using them. This is now fully tested by comparing the specialized kernels against the matrix kernels (or at least they are consistent). To go further, we should add bindings and compare with PennyLane I'd say.